lib/rollbax/item.ex

```diff
@@ -27,7 +27,7 @@ defmodule Rollbax.Item do
     if map_size(meta) == 0 do
       data
     else
-      Map.put(data, "custom", meta)
+      Map.merge(data, meta)
```

Comment:
hey @lexmag,
thanks for responding so fast.

it might break existing code that way, yes. i'm not sure what rollbar does if the custom fields aren't under the `custom` key.

however, as this is just version 0.0.1, that might be fine. i just find it a lot more consistent to put the data in there as the user provides it. especially because there's not only `person` and `request`, but also others like `title`...